### PR TITLE
Fixing "Site search" and "parse_pdb" API

### PR DESCRIPTION
**Pull request by AJK-dev**
Site search (issue #173) - main issue was the replacement of the interaction types and AA - interaction groups
API parse_pdb call - just like the previous GN-assign issue the handling of file uploads in a rest framework request was corrected and function calls were fixed + missing imports added

Additionally there are a few layout refinements (deduplication and consistency) + removal crystal/crystallized in wording

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -756,13 +757,13 @@ class StructureSequenceParser(views.APIView):
...
     parser_classes = (FileUploadParser,)
-    renderer_classes =(JSONRenderer)
+    renderer_classes = (JSONRenderer, )
 
     def post(self, request):
-
-        root, ext = os.path.splitext(request.FILES['pdb_file'].name)
-        header = parse_pdb_header(request.FILES['pdb_file'])
-        parser = SequenceParser(request.FILES['pdb_file'])
+        root, ext = os.path.splitext(request._request.FILES['pdb_file'].name)
```
Comment (esguerra) [inline api/views.py:763]: # root, ext = os.path.splitext(request._request.FILES['pdb_file'].name)

Comment (esguerra) [reply api/views.py:763]: I think this can be commented out?

**Review by esguerra — APPROVED** [review]
We will eventually face a problem with the API since coreAPI got replaced by openAPI in Django3.
For now there's a work around.

https://www.django-rest-framework.org/community/3.10-announcement/

**Comment by AJK-dev** [comment]
> We will eventually face a problem with the API since coreAPI got replaced by openAPI in Django3.
> For now there's a work around.
> 
> https://www.django-rest-framework.org/community/3.10-announcement/

Good note, we should keep this in mind when updating.

File: api/views.py (hunk 1 above)
Comment (AJK-dev) [reply api/views.py:763]: Indeed, just legacy code that I also used for debugging. I've commented them.

**Pull request merged**